assets/styl/_modules/_icon.styl

```diff
@@ -43,9 +43,8 @@
 
     for key, value in $colors
         &-{key}
-            icon-text-color = _contrast-check($color-white, value) ? $color-white : $color-black
             background-color value
-            color icon-text-color
+            color _contrast_choose(value, $color-white $color-black)
```

Comment:
As `$color-white $color-black` is the default value for `$list` argument, shouldn't we write: `_contrast_choose(value)`?